avocado/plugins/spawners/podman.py

```diff
@@ -111,6 +111,34 @@ def __init__(self, config=None, job=None):  # pylint: disable=W0231
         SpawnerMixin.__init__(self, config, job)
         self.environment = f"podman:{self.config.get('spawner.podman.image')}"
 
+    def is_operational(self):
+        podman_bin = self.config.get("spawner.podman.bin")
+        cmd = [podman_bin, "version", "--format={{.Version}}"]
+        try:
+            process = subprocess.Popen(
+                cmd,
+                stdin=subprocess.DEVNULL,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.DEVNULL,
+            )
+        except FileNotFoundError:
+            LOG.error(f"The podman binary {podman_bin} could not be found")
+            return False
+        out, _ = process.communicate()
+        if process.returncode != 0:
+            LOG.error(
+                f"The execution of the podman binary {podman_bin} signaled failure (return code {process.returncode}"
```

Comment:
I would rather break this into something like
```python
            LOG.error( f"The execution of the podman binary {podman_bin} "
                                f"signaled failure (return code {process.returncode}")
```
but I guess that might not be compatible with your current linter.

Reply:
This is now outdated with the reuse of `podman_version`.